src/StreamJsonRpc.Tests/JsonRpcCustomRequestIdTests.cs

```diff
@@ -60,7 +60,7 @@ public JsonRpcWithStringIds(IJsonRpcMessageHandler handler)
 
         protected override RequestId CreateNewRequestId()
         {
-            return new RequestId("prefix-" + base.CreateNewRequestId().Number.Value.ToString());
+            return new RequestId("prefix-" + base.CreateNewRequestId().Number?.ToString());
```

Comment:
💭 Why have the explicit `ToString()`?

```suggestion
            return new RequestId("prefix-" + base.CreateNewRequestId().Number);
```